assets/js/common/ChartDisabledBox/ChartDisabledBox.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function ChartDisabledBox() {
+  return (
+    <div
+      className="mt-4 bg-white shadow rounded-lg py-4 px-8 mx-auto"
+      data-testid="chart-disabled-box"
+    >
+      <h2 className="font-bold text-center text-xl">
+        Charts are disabled, please check documentation for further details
```

Comment:
Maybe `Charts are disabled. Please check the documentation for further details`.
Could we point the user to that documentation with a link?
Otherwise it doesn't look like that helpful

Reply:
Yes, but we don't have the docs yet, so atm I think we could have as is